subsys/bluetooth/audio/has.c

```diff
@@ -868,19 +856,102 @@ static int bt_has_cp_generic_update(struct has_client *client, uint8_t prev_inde
 	}
 }
 
+#if defined(CONFIG_BT_SETTINGS)
+struct client_context_store {
+	/* Last notified preset index */
+	uint8_t last_preset_index_known;
+};
```

Comment:
Should this be packed?

Reply:
Not sure it's needed, as for now there's only 1 byte value there, no padding will be added

Reply:
Right, but you'd be storing a whole word (4 octets), rather than just a single octet this way, right? 

Add `__packed` would be an easy optimization of 75% ;)

Reply:
@MariuszSkamra Should we leave as is or update?

Reply:
The size of this structure is 1. It is because there is no padding.
But just in case, if this structure is about to be extended someday, I'll add `__packed`.
Bear in mind, as it's for flash, the chunks might be written in words anyway (depending on how the flash works).

Reply:
Fair :)